src/main/java/uk/gov/hmcts/sptribs/ciccase/model/CaseData.java

```diff
@@ -239,8 +239,10 @@ public class CaseData {
         label = "Select order template ",
         access = {DefaultAccess.class, CaseworkerWithCAAAccess.class}
     )
+    // private List<ListValue<DraftOrderCIC> draftOrderCIC = new ArrayList<>();
     private DraftOrderCIC draftOrderCIC = new DraftOrderCIC();
 
+
     @JsonUnwrapped(prefix = "link")
```

Comment:
Redundant changes? Don't comment it out, remove it?

Reply:
when you remove the space then build fails.